src/v0/sources/shopify/transform.js

```diff
@@ -170,11 +171,15 @@ const processEvent = async (inputEvent, metricMetadata) => {
   }
   if (message.type !== EventType.IDENTIFY) {
     const anonymousId = await getAnonymousId(message, metricMetadata);
+    const sessionId = await getSessionId(message, metricMetadata);
     if (isDefinedAndNotNull(anonymousId)) {
       message.setProperty('anonymousId', anonymousId);
     } else if (!message.userId) {
       message.setProperty('userId', 'shopify-admin');
     }
+    if (isDefinedAndNotNull(sessionId)) {
+      message.setProperty('sessionId', sessionId);
```

Comment:
I think sessionId is supposed be inside context

Ref from doc: `RudderStack passes the sessionId to the subsequent events in the context.sessionId field.`

Reply:
Okay

Reply:
Done